src/cdk/a11y/aria-describer/aria-describer.ts

```diff
@@ -159,7 +159,7 @@ export class AriaDescriber implements OnDestroy {
       // already a container on the page, but we don't have a reference to it. Clear the
       // old container so we don't get duplicates. Doing this, instead of emptying the previous
       // container, should be slightly faster.
-      if (preExistingContainer) {
+      if (preExistingContainer && preExistingContainer.parentNode) {
         preExistingContainer.parentNode!.removeChild(preExistingContainer);
```

Comment:
You can also remove the `!` after `parentNode` since there's a null check around it.

Reply:
done